src/libs/ReportUtils.js

```diff
@@ -873,8 +873,11 @@ function getDisplayNameForParticipant(login, shouldUseShortForm = false) {
 function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport) {
     return _.map(participants, (participant) => {
         const displayName = getDisplayNameForParticipant(participant.login, isMultipleParticipantReport);
-        const tooltip = participant.login ? Str.removeSMSDomain(participant.login) : '';
 
+        const tooltip = {
+            accountID: participant.accountID,
+        }
```

Comment:
Actually this console error is because avatar souce can in fact be a PropTypes.func, but my proptype incorrectly only accepted strings. Fixed it [here](https://github.com/Expensify/App/pull/20276/commits/8f852403523161a28c62ee7103baadda82b719aa)